home-tasks/src/test/java/com/epam/tc/hw4/ex1/JenkinsExerciseAssertJTest.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.tc.hw4.ex1;
+
+import static com.epam.tc.hw04.component.common.HeaderMenu.HEAD_SERVICE;
+import static com.epam.tc.hw04.component.ex2.ServiceDropdownMenu.HEAD_DIFFERENT_ELEMENTS;
+import static io.qameta.allure.Allure.step;
+
+import com.epam.tc.hw04.pages.DifferentElementsPage;
+import com.epam.tc.hw04.pages.IndexPageForExTwo;
+import com.epam.tc.hw04.support.Data;
+import com.epam.tc.hw4.data.provider.DataProviderTest;
+import com.epam.tc.hw4.steps.SeleniumAbstractCore;
+import com.epam.tc.hw4.storynames.Tags;
+import io.qameta.allure.Description;
+import io.qameta.allure.Epic;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+@Epic(Tags.SUCCESS_TESTS)
+@Feature(Tags.ASSERTJ_TEST_FEATURE)
+public class JenkinsExerciseAssertJTest extends SeleniumAbstractCore {
+
+
+    @Test(description = "Hard asserts test, exercise 1 Jenkins",
+          dataProvider = "testDataForJenkins",
+          dataProviderClass = DataProviderTest.class)
+    @Description("Another path to test by AssertJ")
+    @Story(Tags.INTEGRATION_HARD_STORY)
+    public void checkAndSearchElementsOnDifferentElementsPage(Data data) {
+
+        step("1. Open test site by URL in setUp method");
```

Comment:
Please avoid usage of `io.qameta.allure.Allure.step`, method annotation is enough